src/storage/v2/replication/replication_client.cpp

```diff
@@ -104,9 +104,9 @@ TimestampInfo ReplicationStorageClient::GetTimestampInfo(Storage const *storage)
 
     auto main_time_stamp = storage->repl_storage_state_.last_commit_timestamp_.load();
     info.current_timestamp_of_replica = response.current_commit_timestamp;
-    info.current_number_of_timestamp_behind_master = response.current_commit_timestamp - main_time_stamp;
+    info.current_number_of_timestamp_behind_main = response.current_commit_timestamp - main_time_stamp;
```

Comment:
Shouldn't this be  main_time_stamp- response.current_commit_timestamp;
Because in other case it is negative number?

Reply:
This is just a name change; the logic is the same as it was before.